packages/kbn-esql-validation-autocomplete/src/definitions/options.ts

```diff
@@ -39,7 +39,7 @@ export const metadataOption: CommandOptionsDefinition = {
   validate: (option, command, references) => {
     const messages: ESQLMessage[] = [];
     // need to test the parent command here
-    if (/\[metadata/i.test(command.text)) {
+    if (/metadata/i.test(command.text)) {
```

Comment:
With this change you are creating a bug. Now you are suggesting to the users that if they use brackets it is deprecated while they havent even typed a bracket

<img width="508" alt="image" src="https://github.com/user-attachments/assets/452a498f-5ce6-470a-a16d-69deaf651f6f">

This check is not needed anymore. You need to remove it.